pack/msm_encounter.json

```diff
@@ -0,0 +1,79 @@
+[
+  {
+    "boost": 2,
+    "code": "05025",
+    "faction_code": "encounter",
+    "name": "Home by Dawn",
+    "pack_code": "msm",
+    "position": 25,
+    "quantity": 1,
+    "set_code": "ms_marvel",
+    "text": "<b><i>Give to the Kamala Khan player</i></b>\nYou may flip to your alter-ego form. Choose:\n- Exhaust Kamala Khan → remove Home by Dawn from the game.\n- Discard 1 <b><i>PERSONA</i></b> support you control. If no support was discarded this way, this card gains surge. Discard this obligation.",
+    "type_code": "obligation"
+  },
+  {
+    "base_threat": 2,
+    "boost": 3,
+    "code": "05026",
+    "faction_code": "encounter",
+    "flavor": "<i>The Inventor has convinced the youth of Jersey City that they should serve as human batteries for his experiments</i>",
```

Comment:
Flavor text is italics by default, so no need for tags on it.

Reply:
Updated a new commit